src/s1reader/s1_orbit.py

```diff
@@ -246,7 +246,10 @@ def get_orbit_file_from_dir(zip_path: str, orbit_dir: str, auto_download: bool =
         raise FileNotFoundError(f"{zip_path} does not exist")
 
     if not os.path.isdir(orbit_dir):
-        raise NotADirectoryError(f"{orbit_dir} not found")
+        if not auto_download:
+            raise NotADirectoryError(f"{orbit_dir} not found")
+        else:
+            os.makedirs(orbit_dir, exist_ok=True)
```

Comment:
What do you think about letting the user know that the directory has been created by printing some message to stdout?

Reply:
i've added that print